www/templates/forms.html

```diff
@@ -269,26 +269,10 @@
   </form>
 {% endmacro %}
 
-{% macro rebuild_build(rebuild_url, authz, exactly, ss, builderName, url) %}
+{% macro rebuild_build(rebuild_url, authz, ss, builderName, url) %}
```

Comment:
Can someone confirm that there's no consequences to removing this 'exactly' parameter? I couldn't find rebuild_build being used anywhere else in the code myself.